src/Algebra/Bundles.agda

```diff
@@ -895,3 +1002,17 @@ RawSemigroup = RawMagma
 "Warning: RawSemigroup was deprecated in v1.0.
 Please use RawMagma instead."
 #-}
+
+-- Version 2.0
+
+--Quasigroup = InvertibleMagma
```

Comment:
@MatthewDaggitt  What happens if we are changing the definition in a different version? We cannot use the same name and then the library might have wrong definition.

Reply:
Then we have a problem! We either break compatibility or we are forced to go with a suboptimal name. But all the definitions you've changed were added since v1.7 so I think we're good :smile: